packages/interbit/src/scripts/start.js

```diff
@@ -0,0 +1,59 @@
+// © 2018 BTL GROUP LTD -  This package is licensed under the MIT license https://opensource.org/licenses/MIT
+const {
+  createChains: { createChainsFromConfig },
+  generateDeploymentDetails,
+  startInterbit,
+  setRootChainManifest
+} = require('../chainManagement')
+const { updateIndexHtmls, watchCovenants } = require('../file')
+const { validateConfig } = require('../config')
+
+const start = async options => {
+  const { config, dev, noWatch } = options
+
+  validateConfig(config)
+
+  // TODO: Use options.manifest as initial variable resolution for generating the new one #262
+
+  const { cli } = await startInterbit(undefined, options)
+  // TODO: This create based on config should only run in dev mode, else run chains based on manifest #276
+  const { chainManifest, covenantHashes } = await createChainsFromConfig(
+    cli,
+    config
+  )
+
+  // TODO: This manifest mock is no longer required
+  const deploymentDetails = generateDeploymentDetails(
+    chainManifest,
+    covenantHashes
+  )
+
```

Comment:
I noticed that in `packages/interbit-cli/scripts/start.js` you removed generateDeploymentDetails and a bunch of other cruft. Did you mean to do that here? This file seems to duplicate the old version of that file.

Reply:
This file is functionally the same, cruft included. There are a couple of separate tickets for improving the functionality of start.